code/modules/mob/living/carbon/human/species_types/lizardpeople.dm

```diff
@@ -100,6 +100,7 @@
 	id = SPECIES_ASHWALKER
 	examine_limb_id = SPECIES_LIZARD
 	species_traits = list(MUTCOLORS,EYECOLOR,LIPS, NO_UNDERWEAR)
-	inherent_traits = list(TRAIT_NOGUNS,TRAIT_NOBREATH)
+	inherent_traits = list(TRAIT_NOGUNS,TRAIT_RESISTLOWPRESSURE)
```

Comment:
honestly I think I'd rather them just have the low pressure warning than fully resist 0 pressure, if theres no way to add partial low pressure resistance (its part of /mob/living but I guess its not accessible via species)

Reply:
As far asI know there really isn't a way to do that unless we introduce a new trait. Guess I'll remove it then.

Reply:
There, removed.